hof.settings.json

```diff
@@ -5,6 +5,7 @@
     "hof/components/clear-session",
     "hof/components/session-timeout-warning"
   ],
+  "translations": "./apps/end-tenancy/translations",
```

Comment:
what is this doing? I can run the app without this line and it seems to work fine with the validations, field names, cookie banner etc., is there something else I have missed?